src/Body/FormBody.php

```diff
@@ -150,8 +165,7 @@ private function getMultipartFieldArray(): array
     private function generateMultipartFileHeader(string $name, string $fileName, string $contentType): string
     {
         $header = "Content-Disposition: form-data; name=\"{$name}\"; filename=\"{$fileName}\"\r\n";
-        $header .= "Content-Type: {$contentType}\r\n";
-        $header .= "Content-Transfer-Encoding: binary\r\n\r\n";
```

Comment:
Why did you remove this line?

Reply:
Because afaik Content-Transfer-Encoding is deprecated and about Content-Type I think I readded it?

Reply:
Yeah, not sure whether this is still needed for anyone, but it doesn't really hurt, either.

Reply:
Yeah, true... Indeed I kinda removed it to do some tests, then I saw that it was deprecated so I didn't rebring it there.